releng/raw-binaries/Dockerfile

```diff
@@ -43,11 +49,19 @@ RUN mkdir -p /opt/osxcross && \
 
 ENV PATH="/opt/osxcross/target/bin:${PATH}"
 
-# Wrapper to pull a version of pkg-config at runtime
-ENV PKG_CONFIG=/usr/local/bin/pkg-config.sh
+RUN cd /tmp && \
+    git clone https://github.com/richfelker/musl-cross-make.git && \
+    cd musl-cross-make && \
+    git checkout v0.9.9 && \
+    make MUSL_VER=1.1.24 TARGET=aarch64-unknown-linux-musl DL_CMD="curl -C - -L -o" install && \
+    mv output /opt/musl-cross && \
+    rm -rf /tmp/musl-cross-make
 
-# install go-junit-report for unit tests
-RUN GO111MODULE=on go get github.com/jstemmer/go-junit-report@v0.9.1
+ENV PATH="/opt/musl-cross/bin:$PATH"
+
+# Make sure we have the newer version of pkg-config that works with windows,
+# and install go-junit-report for unit tests
+RUN GO111MODULE=on go get github.com/jstemmer/go-junit-report@v0.9.1 github.com/influxdata/pkg-config@v0.2.7
```

Comment:
I had to use a hard-coded pkg-config version because otherwise we appear to pick up the one in the _flux_ go.mod file, which fails the windows build.